tests/unit/generateCertificateNumber.unitTest.ts

```diff
@@ -0,0 +1,56 @@
+import { TestResultsService } from "../../src/services/TestResultsService";
+import fs from "fs";
+import path from "path";
+import * as dateFns from "date-fns";
+import {cloneDeep} from "lodash";
+
+describe("TestResultsService calling generateExpiryDate", () => {
+    let testResultsService: TestResultsService | any;
+    let MockTestResultsDAO: jest.Mock;
+    let testResultsMockDB: any;
+    let testResultsPostMock: any;
+
+    beforeEach(() => {
+        testResultsMockDB = JSON.parse(fs.readFileSync(path.resolve(__dirname, "../resources/test-results.json"), "utf8"));
```

Comment:
could you not use imports for these? Should already have required tsconfig change.